hw/bsp/fanstel-ev-bt840/pkg.yml

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+pkg.name: hw/bsp/fanstel-ev-bt840
+pkg.type: bsp
+pkg.description: BSP definition for the Fanstel EV-BT840(E)
+pkg.author: "Andrzej Kaczmarek <andrzej.kaczmarek@codecoup.pl>"
```

Comment:
I believe pkg.author has to be "Apache Mynewt <dev@mynewt.apache.org>" for all code contributed like this to mynewt repositories.